instana_client/models/action.py

```diff
@@ -39,14 +39,16 @@ class Action(BaseModel):
     input_parameters: Optional[List[Parameter]] = Field(default=None, description="List of inputs to the action.", alias="inputParameters")
     metadata: Optional[MetaData] = None
     modified_at: datetime = Field(description="Action modified time.", alias="modifiedAt")
-    name: Annotated[str, Field(min_length=1, strict=True, max_length=128)] = Field(description="Action name.")
+    name: Optional[Annotated[str, Field(min_length=1, strict=True, max_length=128)]] = Field(default=None, description="Action name.")
```

Comment:
@Jayssgss All of the code here are generated automatically via openapi generator, does this request any code change for openapi.spec?